debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseConnectorConfig.java

```diff
@@ -584,7 +585,8 @@ public static DecimalHandlingMode parse(String value, String defaultValue) {
                     INCLUDE_SCHEMA_CHANGES,
                     PROPAGATE_COLUMN_SOURCE_TYPE,
                     PROPAGATE_DATATYPE_SOURCE_TYPE,
-                    SNAPSHOT_FULL_COLUMN_SCAN_FORCE)
+                    SNAPSHOT_FULL_COLUMN_SCAN_FORCE,
+                    DatabaseHeartbeatImpl.HEARTBEAT_ACTION_QUERY)
```

Comment:
This is likely not documented in `documentation/modules/ROOT/pages/connectors/mysql.adoc` in the connector options.  Could you please add a row to the options table; you can use the entry from the `postgresql.adoc` as an guide for the same option.